plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -64,7 +64,7 @@ const (
 	// The scheduler uses the annotation to find that the pod shouldn't preempt more pods
 	// when it gets to the head of scheduling queue again.
 	// See podEligibleToPreemptOthers() for more information.
-	NominatedNodeAnnotationKey = "NominatedNodeName"
+	NominatedNodeAnnotationKey = "kubernetes.io/NominatedNodeName"
```

Comment:
I know that in the PR I said we should rename this to `kubernetes.io/nominated-node-name`, but could you please use `scheduler.kubernetes.io/nominated-node-name`? Adding `scheduler` prefix ensures uniqueness and is easier to find the component that has added the annotation.